decision-import-plugin/filter-decision-inputs/README.md

```diff
@@ -0,0 +1,119 @@
+# Optimize decision import plugin - filter out decision variables that you don't want in Optimize
+
+This example demonstrates how to use an Optimize variable plugin to hook into the
+Optimize import and filter out decision inputs you don't want to be added to Optimize. 
+
+After having read through the guide, you will understand
+
+* how the plugin system in Optimize works.
+* how to implement your own Optimize decision import plugin.
+* how to configure the plugin in Optimize.
+
+What is the idea/use case of this demo:
+
+* There are some input values which result to invalid/irrelevant outputs.
+* You don't want to pollute Optimize with unnecessary data.
+
+### How does it work?
+
+To see how the import in Optimize works in general, have a look at the [documentation][4]. 
+The details of the variable import and how the plugin hooks into that 
+are explained in the [decision import plugin readme][5].
+
+### Setup your environment
+
+First, add the Optimize plugin dependency to your project using mavens [pom.xml][3]:
+
+```xml
+<dependency>
+  <groupId>org.camunda.optimize</groupId>
+  <artifactId>plugin</artifactId>
+  <version>${optimize.version}</version>
+</dependency>
+```
+
+To tell maven where to find the plugin environment, add the following repository to your project:
+
+```xml
+<repositories>
+  <repository>
+    <id>camunda-bpm-nexus</id>
+    <name>camunda-bpm-nexus</name>
+    <url>
+      https://app.camunda.com/nexus/content/repositories/camunda-optimize
+    </url>
+  </repository>
+</repositories>
+```
+
```

Comment:
let's add the third party library note as done in the Readme of the output example